marketdata/stream/client_test.go

```diff
@@ -427,7 +427,8 @@ func TestPingFails(t *testing.T) {
 func TestCoreFunctionality(t *testing.T) {
 	connection := newMockConn()
 	defer connection.close()
-	writeInitialFlowMessagesToConn(t, connection, []string{"ALPACA"}, []string{"ALPACA"}, []string{"ALPACA"})
+	writeInitialFlowMessagesToConn(t, connection,
```

Comment:
Please add daily bars to  
`writeInitialFlowMessagesToConn` ("the response from the server") and `checkInitialMessagesSentByClient`

Reply:
`TestSubscripitionAcrossConnectionIssues` should also be updated